scripts/generate/templates/react.template.tsx

```diff
@@ -27,10 +28,10 @@ interface Payload {
  * extension point.
  */
 extend('Checkout::PostPurchase::ShouldRender', async ({storage}) => {
-  const {render, payload} = await getRenderData();
+  const {render, initialState} = await getRenderData();
   if (render) {
-    // Saves payload data, provided to `Render` via `storage.inputData`
-    await storage.update(payload);
+    // Saves initial state, provided to `Render` via `storage.initialData`
```

Comment:
Not sure if it is good or bad to flip back and forth between `state` and `data`. I am finding it tough to wrap my head around so I guess I find it kind of confusing.

Reply:
Two questions:
- Would you like us to rename `storage.initialData` to `storage.initialState`?
- If "no" to the first one, what is your preferred typescript type name in place of `interface InitialState`?

FWIW, I lean toward renaming `initialData` -> `initialState`

Reply:
`storage` kind of makes me think of `data`, not `state`, so I probably slightly prefer sticking to that nomenclature, but I am happy with whichever the team prefers.